### PR TITLE
[Feature] NPCs with bows and arrows do ranged attacks

### PR DESCRIPTION
**Pull request by mackal**
Who knew!!

### CODE REVIEW

**Comment by noudess** [comment]
Cool stuff.

**Comment by Akkadius** [comment]
Anyone test this ?

**Comment by mackal** [comment]
> Anyone test this ?

I tested it.

File: zone/loottables.cpp [hunk 1]
```diff
@@ -400,6 +400,14 @@ void NPC::AddLootDrop(
...
 	item->equip_slot        = EQ::invslot::SLOT_INVALID;
 
+
+	// unsure if required to equip, YOLO for now
+	if (item2->ItemType == EQ::item::ItemTypeBow)
+		SetBowEquipped(true);
+
+	if (item2->ItemType == EQ::item::ItemTypeArrow)
+		SetArrowEquipped(true);
+
```
Comment (Kinglykrab) [inline zone/loottables.cpp:410]: Brackets here maybe?

File: zone/mob_ai.cpp [hunk 2]
```diff
@@ -1337,7 +1337,7 @@ void Mob::AI_Process() {
 			// See if we can summon the mob to us
 			if (!HateSummon()) {
 				//could not summon them, check ranged...
-				if (GetSpecialAbility(SPECATK_RANGED_ATK))
+				if (GetSpecialAbility(SPECATK_RANGED_ATK) || HasBowAndArrowEquipped())
```
Comment (Kinglykrab) [inline zone/mob_ai.cpp:1340]: Brackets.

**Review by Kinglykrab — APPROVED** [review]
👍

**Review by Akkadius — APPROVED** [review]

**Pull request merged**